cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -83,19 +80,21 @@
                 {% if show_categories %}
                     {% import 'macros/category-slug.html' as category_slug %}
                     {% if cat_controls and 'newsroom' in cat_controls.page_type and is_blog(post) %}
-                        {{ category_slug.render('blog', page_url, '', false) }}
+                        {{ category_slug.render(category='blog', href=url) }}
                     {% else %}
                         {% for cat in post.categories.all() %}
                             {% if loop.index > 1 %}
                                 |
                             {% endif %}
-                            {{ category_slug.render(cat.name, page_url, '', false) }}
+                            {{ category_slug.render(category=cat.name, href=url) }}
                         {% endfor %}
                     {% endif %}
                 {% endif %}
             </div>
         </div>
```

Comment:
This section above line 95 (the beginning of the cache) will no longer be cached, thus fixing the issue with the wrong icons displaying (determined by the `category_slug` macro).